src/bp/ui-studio/src/web/components/Layout/Toolbar/ActionItem.tsx

```diff
@@ -0,0 +1,31 @@
+import cx from 'classnames'
+import _ from 'lodash'
+import React from 'react'
+import { OverlayTrigger, Tooltip } from 'react-bootstrap'
```

Comment:
Everytime you see stuff from bootstrap, please remove them, we're trying to get rid of it

Reply:
It's only the tooltips, for now I left the original one to get this PR over with quickly and will change the tooltips for both toolbar and status bar in a different PR